public/index.html

```diff
@@ -26,7 +26,7 @@
       work correctly both with client-side routing and a non-root public URL.
       Learn how to configure a non-root public URL by running `npm run build`.
     -->
-    <title>React App</title>
+    <title>SNEK - Social Network for Engineers and Knowledged</title>
```

Comment:
I think it would be better to have a short page title.
```suggestion
    <title>SNEK</title>
```